docs/program/python-venv.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Prepare your Python Virtual Environment"
+linkTitle: "Prepare your Python Virtualenv for Viam"
```

Comment:
This is quite long in the side nav and also kind of feels odd with that position. It should be the first item, since it's the first thing a user does?

Reply:
Moved up weight

Reply:
```suggestion
linkTitle: "Prepare Virtualenv"
```